api/channels/wecom/channel.py

```
@@ -102,10 +102,32 @@ async def _handle_request(self, request: web.Request) -> web.Response:
+                        message_id,
+                    )
+                    return web.Response(text="")
+                _recent_message_ids[message_id] = now
+                if len(_recent_message_ids) > _MESSAGE_DEDUPE_MAX:
+                    for mid, ts in list(_recent_message_ids.items()):
+                        if now - ts > _MESSAGE_DEDUPE_WINDOW_SECS:
+                            del _recent_message_ids[mid]
+
             try:
-                await channel.handle_decrypted_message(msg)
+                asyncio.create_task(channel.handle_decrypted_message(msg))
```

Ruff (0.16.1)

[warning] Store a reference to the return value of `asyncio.create_task`

(RUF006)